packages/core/integration-tests/test/output-formats.js

```diff
@@ -825,7 +825,7 @@ describe('output formats', function() {
         .find(b => !b.filePath.includes('async'));
       assert(
         workerBundleContents.includes(
-          `importScripts("./${path.basename(syncBundle.filePath)}")`,
```

Comment:
do we still test this case?

Reply:
Yes, there are other tests that cover it